### ISSUE TITLE
Error encountered: No handler found for node kind: Sigmoid (expected: map_sigmoid)

### ISSUE THREAD

**Issue by chiggum** [ethereon/caffe-tensorflow#96]
While converting GoogleNet_SOD_finetune.caffemodel I got this error: "Error encountered: No handler found for node kind: Sigmoid (expected: map_sigmoid)"

The last layer in my network prototxt is a Sigmoid layer.

**Comment by kendemu** [comment]
I'm also stuck with the same problem.
By reading the code, I found that caffe-tensorflow only implemented Softmax Layer for FCN.
Use softmax or you have to define def map_sigmoid() in kaffe/tensorflow/transformer.py and define def sigmoid() in kaffe/tensorflow/network.py

**Comment by xuyinbo** [comment]
@kendemu I encountered with the same problem with @chiggum. So can you tell me how to define def map_sigmoid() in kaffe/tensorflow/transformer.py and define def sigmoid() in kaffe/tensorflow/network.py? Thanks a lot!

**Comment by kendemu** [comment]
@xuyinbo @chiggum 
In Network class from [kaffe/tensorflow/network.py](https://github.com/ethereon/caffe-tensorflow/blob/master/kaffe/tensorflow/network.py), implement method sigmoid.
The transformer.py have an ability to do reflection of the Network class methods, so you don't have to modify transformer.py.
e.g. : softmax -> map_softmax
``` python
    @layer
    def sigmoid(self, input, name):
	return tf.nn.sigmoid(input, name=name)
```
You can see the implementation of softmax for the implementation sample.
